app/controllers/admin/attachments_controller.rb

```diff
@@ -188,4 +192,9 @@ def save_attachment
       end
     end
   end
+
+  def run_updater
```

Comment:
I would call this `update_attachments` and I reckon we don't need to memoise the variable we can just do:

```rb
ServiceListeners::AttachmentUpdater.new(attachable.reload).update!
```